compiler/optimizer/GlobalRegisterAllocator.cpp

```diff
@@ -350,20 +350,6 @@ TR_GlobalRegisterAllocator::perform()
    comp()->getOptimizer()->setSeenBlocksGRA(0);
    comp()->getOptimizer()->setSuccessorBitsGRA(0);
 
-   if (comp()->useCompressedPointers() &&
-       comp()->cg()->materializesHeapBase() &&
-       TR::Compiler->vm.heapBaseAddress() != 0)
-      {
-      TR::SymbolReference *autoSymRef = NULL;
-      vcount_t visitCount = comp()->incVisitCount();
-      for (TR::TreeTop * tt = comp()->getStartTree(); tt; tt = tt->getNextTreeTop())
-         {
-         changeHeapBaseConstToLoad(comp(), autoSymRef, tt->getNode(), visitCount);
```

Comment:
As far as I can see if you eliminate the call to `changeHeapBaseConstToLoad` here then this function is dead. Can we remove it?